tslint.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
we should pick either `eslint` or `tslint`.

Reply:
es-lint it is